scipy/stats/tests/test_multivariate.py

```diff
@@ -447,15 +604,38 @@ def test_rvs_shape(self):
         sample = multivariate_normal.rvs(mean=np.zeros(d), cov=1, size=N)
         assert_equal(sample.shape, (N, d))
 
+        sample = multivariate_normal.rvs(
+            np.zeros(d), cov=None, inverse_cov=1, size=N)
+        assert_equal(sample.shape, (N, d))
+
         sample = multivariate_normal.rvs(mean=None,
                                          cov=np.array([[2, .1], [.1, 1]]),
                                          size=N)
         assert_equal(sample.shape, (N, 2))
 
+        inv_cov = np.array([[3.6, 0.2], [4.4, 10.2]])
+        sample = multivariate_normal.rvs(
+            None, cov=None, inverse_cov=inv_cov, size=N)
+        assert_equal(sample.shape, (N, 2))
+
         u = multivariate_normal(mean=0, cov=1)
         sample = u.rvs(N)
         assert_equal(sample.shape, (N, ))
 
+        u = multivariate_normal(mean=0, cov=None, inverse_cov=1)
+        assert_equal(u.rvs(N).shape, (N,))
+
+        # We currently always squeeze the output.
+        sample = multivariate_normal.rvs(
+            mean=None, cov=np.identity(3), size=(1, 2)
+        )
+        assert_equal(sample.shape, (2, 3))
+
+        sample = multivariate_normal.rvs(
+            mean=None, cov=None, inverse_cov=np.identity(3), size=(1, 2)
+        )
+        assert_equal(sample.shape, (2, 3))
+
```

Comment:
We won't add `rvs` in the initial PR, but maybe we can bring these tests back when we do.